app/views/announcements/_ind_announcement_admin.html.erb

```diff
@@ -2,7 +2,7 @@
 
   <div class='col3 right-align'>
     <div class='column-inner'>
-      <div class='announcement-picture'>
+      <div class='show-picture'>
```

Comment:
Maybe for the next PR you can use the `announcement-picture` class and add css adjustments accordingly! `show-picture` is namespaced for a reason, so doesn't _really_ make sense to use it here, although seems like this works :)